lib/vsc/utils/fancylogger.py

```diff
@@ -345,6 +345,8 @@ def deprecated(self, msg, cur_ver, max_ver, depth=2, exception=None, *args, **kw
             self.raiseException("DEPRECATED (since v%s) functionality used: %s" % (max_ver, msg), exception=exception)
         else:
             deprecation_msg = "Deprecated functionality, will no longer work in v%s: %s" % (max_ver, msg)
+            if verbose:
+                sys.stderr.write(deprecation_msg)
```

Comment:
Well, we don't use `log.error` in EasyBuild (we raise an exception instead), and `log.warning` is used for a bunch of things that may signal a problem but usually fix themselves later (e.g. not being able to resolve templates at an early stage)

I guess it boils down to semantics, deprecation warnings are another 'level' of warnings to me.